src/views/perms/const.js

```diff
@@ -33,10 +34,14 @@ export const UserAssetPermissionListPageSearchConfigOptions = [
 
 export const AssetPermissionTableMeta = {
   name: {
+    formatter: DetailFormatter,
     minWidth: '120px',
     formatterArgs: {
       route: 'AssetPermissionDetail',
-      drawer: true
+      drawer: true,
+      routeQuery: {
+        tab: 'Basic'
+      }
     }
   },
   action: {
```

Comment:
This is a snippet of a Vue.js component, specifically part of the page called "UserAssetPermissionListPage" that deals with user asset permission list. The purpose appears to be finding ways for users who have different access levels (such as admin or manager) to view specific information related to assets in their company's systems.

From this snippet:

* It contains references to `i18n` and `AmountFormatter`, indicating it provides localization functionality for translation across languages.
* There isn't an explicit function mentioned; however, since we're focusing on potential issues, there might not be any critical problems currently detected.

However, if you were referring to more than just these two components (`AmountFormatter` and `DetailViewer`) then please share those details.